aqa_5/src/ui/pages/products/deleteProduct.page.ts

```diff
@@ -0,0 +1,22 @@
+import { SalesPortalPage } from "../salesPortal.page"
+
+class DeleteProductPage extends SalesPortalPage {
```

Comment:
```suggestion
class DeleteProductModal extends SalesPortalPage {
```
Все таки это модалка, а не страница)